_includes/api/en/4x/express.static.md

```diff
@@ -3,7 +3,7 @@
 This is a built-in middleware function in Express.
 It serves static files and is based on  [serve-static](/resources/middleware/serve-static.html).
 
-<div class="doc-box doc-info" markdown="1">NOTE: For best results, [use a reverse proxy](/{{page.lang}}/advanced/best-practice-performance.html#use-a-reverse-proxy) cache to improve performance of serving static assets.
+<div class="doc-box doc-info" markdown="1">For best results, [use a reverse proxy](/{{page.lang}}/advanced/best-practice-performance.html#use-a-reverse-proxy) cache to improve performance of serving static assets.
```

Comment:
Why aren't we using the admonition template here?

Reply:
Because at the time of Jekyll transforming the link it was giving me problems, I have the solution but I don't entirely like it.

Reply:
done

Reply:
I'm not crazy about it, either, but for now I think it's the best solution, thanks.